src/state/farms/hooks.ts

```diff
@@ -13,6 +13,18 @@ import { State, Farm, FarmsState } from '../types'
 export const usePollFarmsData = (includeArchive = false) => {
```

Comment:
Let's rename this one `usePollFarmsPublicData`?

Reply:
Done